### ISSUE TITLE
IPV6 Support

### ISSUE THREAD

**Issue by ghost** [bristolcrypto/SPDZ-2#65]
It seems like there is no IPV6 Support based on the usage of getaddrinfo in sockets.cpp?

https://github.com/bristolcrypto/SPDZ-2/blob/master/Networking/sockets.cpp#L109
https://github.com/bristolcrypto/SPDZ-2/blob/master/Networking/sockets.cpp#L118

**Comment by mkskeller** [comment]
Indeed, we haven't ever considered this. If you are intending to change this, I'd suggest you do so either in SCALE-MAMBA or MP-SPDZ because this project isn't under active development any more.
